src/Microsoft.DncEng.SecretManager/StorageTypes/AzureKeyVault.cs

```diff
@@ -51,19 +54,20 @@ public override async Task<List<SecretProperties>> ListSecretsAsync(AzureKeyVaul
         var secrets = new List<SecretProperties>();
         await foreach (var secret in client.GetPropertiesOfSecretsAsync())
         {
-            DateTimeOffset nextRotationOn = GetNextRotationOn(secret.Tags);
+            DateTimeOffset nextRotationOn = GetNextRotationOn(secret.Name, secret.Tags);
             ImmutableDictionary<string, string> tags = GetTags(secret);
             secrets.Add(new SecretProperties(secret.Name, secret.ExpiresOn ?? DateTimeOffset.MaxValue, nextRotationOn, tags));
         }
 
         return secrets;
     }
 
-    private static DateTimeOffset GetNextRotationOn(IDictionary<string, string> tags)
+    private DateTimeOffset GetNextRotationOn(string name, IDictionary<string, string> tags)
     {
         if (!tags.TryGetValue(_nextRotationOnTag, out var nextRotationOnString) ||
             !DateTimeOffset.TryParse(nextRotationOnString, out var nextRotationOn))
         {
+            _console.LogError($"Key Vault Secret '{name}' is missing {_nextRotationOnTag} tag, using the end of time as value. Please force a rotation or manually set this value.");
```

Comment:
Instead of just saying "force a rotation", it'd be useful to say "rerun with --force-secret={name}"

Reply:
Thats not exactly doable. `--force-secret={name}` doesn't work in all cases.

This is the actual name of a value in the vault, and it may be a part of a composite secret, like a github app identity.

Reply:
The thing that actually needs to be --force-secreted is the higher level entry in the manifest, not the single value in the vault.